packages/shared/src/hooks/useInAppNotification.ts

```diff
@@ -0,0 +1,60 @@
+import { useMemo } from 'react';
+import { useQuery, useQueryClient } from 'react-query';
+import {
+  NewNotification,
+  NEW_NOTIFICATIONS_SUBSCRIPTION,
+} from '../graphql/notifications';
+import useSubscription from './useSubscription';
+
+interface UseInAppNotification {
+  displayNotification: (
+    notification: NewNotification,
+    params?: NotifyOptionalProps,
+  ) => void;
+  dismissNotification: () => void;
+}
+
+export interface IInAppNotification {
```

Comment:
The double I at the beginning can be confusing

Reply:
It was a common practice to prepend the interface name with an `I` in my previous company, I can rename it if you like

Reply:
Updated 👍

Reply:
We usually don't this practice at this point. For me the only concern is the double I. I didn't realize for a bit that they are different types. Felt like a typo